packages/taquito/src/operations/batch-operation.ts

```diff
@@ -33,6 +33,19 @@ export class BatchOperation
     }, 0);
   }
 
+  public getOriginatedContractAddresses(): string[] {
+    const originationOpResults = this.results.filter(
+      (x) => x.kind === 'origination'
+    ) as OperationContentsAndResultOrigination[];
+
+    let addresses: string[] = [];
+    for (const res of originationOpResults) {
+      addresses = [...addresses, ...res.metadata.operation_result.originated_contracts!];
```

Comment:
Are we sure that `originated_contracts` will always be defined? Could it lead to an unexpected error if it is not the case? It might be safer to validate that originated_contracts is defined and then append its values rather than using the non-nul assertion operator.

Reply:
that's a good point, I was under the impression that the property will always be there. I'll amend it